cfy_manager/components/execution_scheduler/config/cloudify-execution-scheduler

```diff
@@ -0,0 +1,4 @@
+# REST Service's config files
+MANAGER_REST_CONFIG_PATH="{{ restservice.home_dir }}/cloudify-rest.conf"
+
+LOG_PATH="{{ execution_scheduler.log_dir }}/execution_scheduler.log"
```

Comment:
@glukhman do we really need these ? do not think so, usually we have configuration file for systemd and I do not think so we are going to run on systemd on 5.2 ?

@tehasdf what do you think?